addons/sale_management/models/sale_order.py

```diff
@@ -88,8 +88,9 @@ def _compute_option_data_for_template_change(self, option):
     def update_prices(self):
         self.ensure_one()
         res = super().update_prices()
+        # TODO env.add_to_compute after onchange conversion to computes
         for line in self.sale_order_option_ids:
-            line.price_unit = self.pricelist_id.get_product_price(line.product_id, line.quantity, self.partner_id, uom_id=line.uom_id.id)
+            line.price_unit = self.pricelist_id._get_product_price(line.product_id, line.quantity, uom=line.uom_id)
```

Comment:
self.sale_order_option_ids._compute_price_unit()
self.sale_order_option_ids._compute_discount()

Reply:
Also remove the
```python
        if not is_html_empty(template.note):
            self.note = template.note
```

below in the onchange, it is managed by the compute